### ISSUE TITLE
Variable renaming might not work across data series

### ISSUE THREAD

**Issue by josephholler** [walkerke/tidycensus#319]
I am trying to download two race/ethnicity variables and two poverty variables from ACS 2018 as follows:

`bronxTracts <- get_acs(state="NY", county="005", geography="tract", variables=c(pop_re="DP05_0070", white="DP05_0077", pop_ps="S1701_C01_001", belpov="S1701_C02_001"), year=2018, geometry=TRUE, output="wide", key="...")`

The output `str()` is:

```
Classes ‘sf’, ‘tbl_df’, ‘tbl’ and 'data.frame':	339 obs. of  13 variables:
 $ GEOID         : chr  "36005000100" "36005000200" "36005000400" "36005001600" ...
 $ NAME          : chr  "Census Tract 1, Bronx County, New York" "Census Tract 2, Bronx County, New York" "Census Tract 4, Bronx County, New York" "Census Tract 16, Bronx County, New York" ...
 $ S1701_C01_001E: num  0 4542 5624 5674 2604 ...
 $ S1701_C01_001M: num  17 574 514 441 314 1040 323 247 552 301 ...
 $ S1701_C02_001E: num  0 1028 549 1264 843 ...
 $ S1701_C02_001M: num  17 396 249 449 255 945 467 12 525 360 ...
 $ DP05_0070E    : num  7080 4542 5634 5917 2765 ...
 $ DP05_0070M    : num  290 574 517 461 307 ...
 $ DP05_0077E    : num  538 77 209 220 262 110 1 69 95 0 ...
 $ DP05_0077M    : num  160 72 88 120 91 92 3 111 67 12 ...
 $ geometry      :sfc_MULTIPOLYGON of length 339; first list element: List of 1
...
```

However, if I use only `variables=c(pop_re="DP05_0070", white="DP05_0077")` or `variables=c(pop_ps="S1701_C01_001", belpov="S1701_C02_001")`, the variable names are `pop_re`, `white`, `pop_ps`, and `belpov` as expected. I'm guessing it may be a problem that the two pairs of variables are coming from profile tables and subject tables, respectively?

**Comment by mfherman** [comment]
Hi @josephholler and thanks for the report. You are indeed right that some of the logic we added in #214 and #215 borked the ability to pass custom names to the variables. I think this should be a pretty easy fix when I have a chance.

For now, as you've noticed if you make separate calls for each data series (B/C, S, DP) the renaming will work.

**Comment by josephholler** [comment]
Ok! Thanks for a quick response!

**Comment by mfherman** [comment]
Seems like this was a simple one-line fix. I've opened up a PR with the change, so it should get merged pretty soon. When it is merged, you will need to install the GitHub version of tidycensus for it to work (until a new version is pushed to CRAN).

Here is a working example from my machine:

```r
get_acs(
  state = "NY",
  county = "005",
  geography = "tract",
  variables = c(
    pop_re = "DP05_0070",
    white = "DP05_0077",
    pop_ps = "S1701_C01_001",
    belpov = "S1701_C02_001"
  ),
  year = 2018,
  output = "wide"
)
# Getting data from the 2014-2018 5-year ACS
# Fetching data by table type ("B/C", "S", "DP") and combining the result.
# # A tibble: 339 x 10
#    GEOID       NAME                                       pop_psE pop_psM belpovE belpovM pop_reE pop_reM whiteE whiteM
#    <chr>       <chr>                                        <dbl>   <dbl>   <dbl>   <dbl>   <dbl>   <dbl>  <dbl>  <dbl>
#  1 36005000100 Census Tract 1, Bronx County, New York           0      17       0      17    7080     290    538    160
#  2 36005000200 Census Tract 2, Bronx County, New York        4542     574    1028     396    4542     574     77     72
#  3 36005000400 Census Tract 4, Bronx County, New York        5624     514     549     249    5634     517    209     88
#  4 36005001600 Census Tract 16, Bronx County, New York       5674     441    1264     449    5917     461    220    120
#  5 36005001900 Census Tract 19, Bronx County, New York       2604     314     843     255    2765     307    262     91
#  6 36005002000 Census Tract 20, Bronx County, New York       9394    1040    4207     945    9409    1043    110     92
#  7 36005002300 Census Tract 23, Bronx County, New York       4599     323    2187     467    4600     323      1      3
#  8 36005002400 Census Tract 24, Bronx County, New York        172     247       0      12     172     247     69    111
#  9 36005002500 Census Tract 25, Bronx County, New York       5874     552    2339     525    5887     553     95     67
# 10 36005002701 Census Tract 27.01, Bronx County, New York    2868     301    1642     360    2868     301      0     12
# # ... with 329 more rows
```

Thanks again for the bug report, @josephholler !

**Comment by walkerke** [comment]
Merged #322 and can confirm that this now works as expected in the development version (0.10.3).
